docs/accounting.md

```diff
@@ -0,0 +1,26 @@
+# Accounting Service
+
+The design of the accounting service for Advanca is aimed at providing an accurate reporting of the services provided. Specifically, the accounting service has to protect both the user and worker against each other from malicious tampering and mis-reporting.
+
+**There are several limitations that we have to work around:**
+
+* There is no trusted time service within Intel SGX enclaves
+* There is no trusted way to measure elapsed time or cpu cycles in SGX1
+  * RDTSC is available in SGX2There is no trusted IO available for the enclave
+
+  **Accounting information which we are interested in:**
```

Comment:
Is the indentation here intended? The [rendered page](https://github.com/advanca/advanca/blob/dfd6eb432dc89df8c7be8d8800d326491caf0dec/docs/accounting.md) looks a bit odd.